chroma-manager/tests/framework/utils/install_deps.sh

```diff
@@ -4,4 +4,5 @@ mkdir -p chroma_test_env
 tar -C chroma_test_env -xizf -"
 
 ssh root@$CHROMA_MANAGER "set -ex
-yum -y install nodejs npm nginx libuv"
+yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/managerforlustre/manager-for-lustre/repo/epel-7/managerforlustre-manager-for-lustre-epel-7.repo
+yum -y install nodejs npm nginx libuv iml-gui iml-srcmap-reverse"
```

Comment:
_iml-gui_ and _iml-srcmap-reverse_ are supposed to get installed when the manager is installed through the _Requires:_ additions to the _chroma-manager.spec_ changes above.

But that should happen in production environments also.  Adding the Copr repo for those packages in this file only adds it in the test environment.

What you actually [need](https://github.com/intel-hpdd/intel-manager-for-lustre/pull/142/files#diff-ac0571f67c3363c9e3acdc025d81fe71R366) is in another [WiP patch](#142).